pyspectral/blackbody.py

```diff
@@ -99,24 +99,17 @@ def planck(wave, temperature, wavelength=True):
         LOG.debug("Using {0} when calculating the Blackbody radiance".format(
             units[(wavelength is True) - 1]))
 
-    if np.isscalar(temperature):
-        temperature = np.array([temperature, ], dtype='float64')
-    elif isinstance(temperature, (list, tuple)):
-        temperature = np.array(temperature, dtype='float64')
-
+    temperature = _scalar_tuple_list_to_numpy(temperature)
     shape = temperature.shape
-    if np.isscalar(wave):
-        wln = np.array([wave, ], dtype='float64')
-    else:
-        wln = np.array(wave, dtype='float64')
+    wave = _scalar_tuple_list_to_numpy(wave)
```

Comment:
In #203, we chose to just ignore this and only allow for arrays and scalars to be passed, and skip/delete the conversion code entirely

Reply:
Tuple/list support removed in https://github.com/pytroll/pyspectral/pull/206/commits/17e62ac93fdc6a8279445345640d8ac69e711fd0